core/templates/pages/admin-page/admin-page.directive.ts

```diff
@@ -49,12 +49,12 @@ angular.module('oppia').directive('adminPage', [
         '/pages/admin-page/admin-page.directive.html'),
       controllerAs: '$ctrl',
       controller: [
-        '$location', '$rootScope', '$scope', 'AdminDataService',
-        'AdminRouterService', 'CsrfTokenService', 'PlatformFeatureService',
+        '$location', '$rootScope', '$scope', 'AdminRouterService',
+        'CsrfTokenService', 'PlatformFeatureService', 'UserBackendApiService',
```

Comment:
Do we need to import/require the file where UserBackendApiService is defined?

Reply:
I'm sorry I don't get it.